py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -1579,11 +1579,9 @@ def test_convert_time_zone_lazy_schema() -> None:
 
 def test_convert_time_zone_on_tz_naive() -> None:
     ts = pl.Series(["2020-01-01"]).str.strptime(pl.Datetime)
-    with pytest.raises(
-        ComputeError,
-        match="cannot call `convert_time_zone` on tz-naive; set a time zone first with `replace_time_zone`",
-    ):
-        ts.dt.convert_time_zone("Africa/Bamako")
```

Comment:
`Africa/Bamako` has a +00:00 offset, so I'm changing the test to use a more unusual one (doesn't get much more unusual than +05:45)